app/ui/src/main/java/de/mm20/launcher2/ui/launcher/SharedLauncherActivity.kt

```diff
@@ -132,6 +134,15 @@ abstract class SharedLauncherActivity(
                             val reverseSearchResults by viewModel.reverseSearchResults.observeAsState(false)
                             val fixedSearchBar by viewModel.fixedSearchBar.observeAsState(false)
 
+                            viewModel.fixedRotation.observe(this) { fixedRotation ->
+                                requestedOrientation = if (fixedRotation) {
+                                    ActivityInfo.SCREEN_ORIENTATION_PORTRAIT + ActivityInfo.SCREEN_ORIENTATION_NOSENSOR
```

Comment:
Assuming here that `SCREEN_ORIENTATION`-values work as flags

Reply:
I think so but please use bitwise or (`or`) to combine the flags

Reply:
Well, that did not work anyway. :)